content/posts/0022-bionic-eos.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Bionic End of Support
+date: "2023-07-28"
+slug: bionic-eos
+author: swigmore
+---
+
+As of May 31, 2023, Ubuntu 18.04 (Bionic Beaver) is [out of
+support](https://ubuntu.com/blog/18-04-end-of-standard-support) for open-source
+customers.
+
+As a result, Paketo will stop supporting Bionic in the project. If you
+currently depend on our Bionic-based offerings, this blog post will contain
+all of the relevant information about what this means and how to migrate.
+
+## Deprecation Details
+
+The removal of Bionic support in Paketo was announced and ratified in the
+[Bionic End of Support RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0057-bionic-eos.md).
+The following stacks and builders will no longer be supported, meaning
+  - (1) we will no longer publish updates (meaning no CVE fixes),
+  - (2) the related repositories have been archived, and
+  - (3) the images will be removed from Docker and GCR in accordance with our [image retention policy](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0046-image-retention-policy.md)
+
+
+  * [Bionic Tiny Stack](https://github.com/paketo-buildpacks/bionic-tiny-stack)
+  * [Bionic Tiny Builder](https://github.com/paketo-buildpacks/tiny-builder)
+  * [Bionic Tiny Buildpackless Builder](https://github.com/paketo-buildpacks/buildpackless-tiny-builder)
+
+  * [Bionic Base Stack](https://github.com/paketo-buildpacks/bionic-base-stack)
+  * [Bionic Base Builder](https://github.com/paketo-buildpacks/base-builder)
+  * [Bionic Base Buildpackless Builder](https://github.com/paketo-buildpacks/buildpackless-base-builder)
+
+  * [Bionic Full Stack](https://github.com/paketo-buildpacks/bionic-full-stack)
+  * [Bionic Full Builder](https://github.com/paketo-buildpacks/full-builder)
+  * [Bionic Full Buildpackless Builder](https://github.com/paketo-buildpacks/buildpackless-full-builder)
+
+## Migration Path
+
+We strongly recommend migrating your workloads from Bionic to our Ubuntu 22.04
+(Jammy Jellyfish) based offerings. Each Bionic stack and builder listed above
+has a more or less 1:1 Jammy equivalent available in the project.
+
+If you are using the `pack` CLI version v0.29.0 or prior for your builds, you
+can set the default builder for your builds to Jammy via the `pack config default-builder` command:
+```
+pack config default-builder paketobuildpacks/builder-jammy-base
+```
+In future versions of `pack`, Jammy will be the default.
```

Comment:
I don't think this is true. Pack doesn't have a default builder. You always need to set a default builder, if you want a default to be applied.

What will be updated in the next pack release is the output from `pack builder suggest`. Presently, it still suggests the EOL builders. We are waiting on upstream to release pack so it suggests the correct things.

Reply:
```suggestion
You may notice that current versions of `pack` still suggest using the Bionic builder if you run `pack builder suggest`. This is out-of-date and Paketo is waiting on a new release of `pack` which will correctly suggest the Jammy builders. Please ignore the output of `pack builder suggest` for now and use the builder listed above.
```